stable/burrow/templates/service-peers.yaml

```diff
@@ -1,25 +1,36 @@
+{{- $peerIngress := and .Values.peer.ingress.enabled (not (eq (len .Values.peer.ingress.hosts) 0)) -}}
 {{ range (until (.Values.chain.numberOfNodes | int)) }}
 {{- $nodeNumber := printf "%03d" . }}
 ---
 kind: Service
 apiVersion: v1
 metadata:
+{{- if $peerIngress }}
+  annotations:
+    external-dns.alpha.kubernetes.io/hostname: "{{ $nodeNumber }}.{{ index $.Values.peer.ingress.hosts 0 }}"
```

Comment:
I'm wondering if we're too opinionated here? Probably not, but maybe something to consider.